website/profile/views.py

```diff
@@ -521,12 +529,44 @@ def update_mailchimp_subscription(user, list_name, subscription, send_goodbye=Tr
             pass
     else:
         try:
-            mailchimp_utils.unsubscribe_mailchimp_async(list_name, user._id, username=user.username, send_goodbye=send_goodbye)
+            mailchimp_utils.unsubscribe_mailchimp_async(list_name, user._id, username=user.username)
         except (MailChimpError, OSFError):
             # User has already unsubscribed, so nothing to do
             pass
 
 
+@user_account_merged.connect
+def send_account_merged_message(user):
+    """ Sends a message using Celery messaging to alert other services that an osf.io user has been merged."""
+    publish_merged_user(user)
+
+
+@user_account_merged.connect
+def unsubscribe_old_merged_account_from_mailchimp(user):
+    """ This is a merged account (an old account that was merged into an active one) so it needs to be unsubscribed
+    from mailchimp."""
+    if not settings.RUNNING_MIGRATION:
```

Comment:
hmm, have we completed [the modm to django migration](https://github.com/CenterForOpenScience/osf.io/blob/c9774718fdaf028d1d123c27d6b82970d238904d/website/settings/defaults.py#L769) yet?